app/assets/stylesheets/views/_coming-soon.scss

```diff
@@ -0,0 +1,4 @@
+.summary {
```

Comment:
This will effect all `.summary` elements in the application rather than just the coming soon ones. The way we try and do this is to use the `page_class` as the root selector in the view Sass file and then nest everything inside that selector. You can see this in [`views/_case-study.scss`](https://github.com/alphagov/government-frontend/blob/coming_soon/app/assets/stylesheets/views/_case-studies.scss) where the outermost selector is `.case-studies {` and all the page elements are nested within it. Hope that makes sense.